services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -85,6 +85,28 @@ function hasIdColumn(fields, primaryKeys) {
     || _.includes(primaryKeys, 'id');
 }
 
+function isJunctionTable(fields, constraints) {
+  const FIELDS_TO_IGNORE = [
+    'createdAt', 'updatedAt', 'deletedAt',
+    'creationDate', 'updateDate', 'deleteDate',
+  ];
```

Comment:
The way we detect technical timestamps is error prone IMO.
We don't even check the `type` of the columns.

Reply:
I don't think so but tell me in which case it could ?

Reply:
you cannot anticipate all the technical timestamps name you can have (coming from custom database column names, from ORM migrations, from database conventions...)
For instance `touchAt`, `touchedAt`, `cDate`, `cAt`, `created`, `dateCreation`,... could be technical timestamp too.

Never mind, we'll iterate on this if issues occur.